tests/test_result.py

```diff
@@ -5,6 +5,31 @@
 from qibo.result import CircuitResult, MeasurementOutcomes, load_result
 
 
+@pytest.mark.parametrize("qubits", [[0, 2], [0], [1, 2]])
+def test_measurementoutcomes_probabilties(backend, qubits):
+    c = models.Circuit(3)
+    c.add(gates.X(0))
+    c.add(gates.M(0, 2))
+    global_probs = c().probabilities(qubits=[0, 2])
+    probabilities = c().probabilities(qubits=qubits)
+    c.has_collapse = True
+    if 1 in qubits:
+        with pytest.raises(Exception) as exc_info:
+            repeated_probabilities = c().probabilities(qubits=qubits)
+            assert (
+                str(exc_info.value)
+                == "Asking probabilities for qubits that were not measured."
+            )
```

Comment:
```suggestion
        with pytest.raises(RuntimeError):
            repeated_probabilities = backend.execute_circuit(c).probabilities(qubits=qubits)
```

Reply:
Like my comment about `raise_error`, this should be unified too.

Reply:
Another point about standardization, everywhere else `self` is used but here `cls` is being used.